src/engine/rendertext.cpp

```diff
@@ -341,11 +341,11 @@ static float icon_width(const char *name, float scale)
     { \
         h++; \
         bool alt = textblinking && totalmillis%(textblinking*2) > textblinking; \
-        if(s) TEXTCOLOR(h); \
+        if(s) TEXTCOLOR(h) {} \
```

Comment:
why `TEXTCOLOR(h) {}` and not `{ TEXTCOLOR(h) }` ? (is repeated after too)

Reply:
oh *** good catch